sky/cli.py

```diff
@@ -1284,6 +1285,7 @@ def exec(
     zone: Optional[str],
     workdir: Optional[str],
     gpus: Optional[str],
+    cpus: Optional[str],
```

Comment:
If we are ignoring it, may be better to not expose `--cpus` in `sky exec`? IIUC, the main motivation is to make sure `sky launch -c existing --cpus x` works.

Reply:
You're right. It was my misunderstanding. My intention was to allow `sky exec existing-cluster x.yaml` when `x.yaml` includes CPU requirements. I thought this requires adding `cpus` CLI argument in `sky exec`, which isn't actually true.

Now this is fixed; `cpus` is included in `sky exec` CLI args, but can be in the YAML file. PTAL.